osu.Game/Overlays/UserProfileOverlay.cs

```diff
@@ -26,7 +26,7 @@ public class UserProfileOverlay : FullscreenOverlay
         private ProfileSectionsContainer sectionsContainer;
         private ProfileSectionTabControl tabs;
 
-        public const float CONTENT_X_MARGIN = 70;
+        public const float CONTENT_X_MARGIN = 50;
```

Comment:
this change is not valid. the user profile design still uses 70px padding, and rankings use 50px. you can't conflate two constants (especially in two disparate overlay namespaces) like this.